python/python/glide/constants.py

```diff
@@ -108,3 +108,13 @@
         ],
     ],
 ]
+
+FtSearchResponse = List[
+    Union[int, Mapping[TEncodable, Mapping[TEncodable, TEncodable]]]
+]
+
+FtAggregateResponse = List[Mapping[TEncodable, Any]]
+
+FtProfileResponse = List[
+    Union[FtSearchResponse, FtAggregateResponse, Mapping[TEncodable, Any]]
```

Comment:
not knowing much about FT.PROFILE, could we change this to:
```suggestion
    Union[FtSearchResponse, FtAggregateResponse, Mapping[string, int]]
```

Reply:
see: https://github.com/valkey-io/valkey-glide/blob/release-1.2/java/client/src/main/java/glide/api/commands/servermodules/FT.java#L500

Reply:
Not sure about JAVA, but the example returns byte.